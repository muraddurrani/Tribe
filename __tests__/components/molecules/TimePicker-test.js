import React from 'react';
import renderer from 'react-test-renderer';
import TimePicker from '../../../src/components/molecules/Hometile'

jest.useFakeTimers()

it('TimePicker renders correctly', () => {
  const tree = renderer
    .create(
    <TimePicker 
      label = 'Appointment Time' 
      placeholder = '03:00pm' 
      onSelect = {console.log('Time selected')}
      />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});