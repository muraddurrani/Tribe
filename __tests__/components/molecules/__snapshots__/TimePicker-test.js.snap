// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePicker renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fdfdfd",
      "borderRadius": 10,
      "width": "90%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "height": 50,
        "paddingHorizontal": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#242424",
          "fontSize": 16,
          "marginLeft": 10,
        }
      }
    >
      
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#f7f7f7",
        "borderBottomLeftRadius": 10,
        "borderBottomRightRadius": 10,
        "borderTopColor": "#ededed",
        "borderTopWidth": 1,
      }
    }
  />
</View>
`;
